src/app/App.jsx

```diff
@@ -1,71 +1,31 @@
 import React from 'react';
-import logo from '../logo.svg';
-import { Counter } from '../features/counter/Counter';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+} from 'react-router-dom';
 import './App.scss';
+import Home from '../components/pages/PageHome';
+import Signin from '../components/pages/PageSignin';
```

Comment:
I think it should be `SignIn` with a capital i.

Reply:
Done. Also changed the file name to PageSignIn.jsx to keep consistent case structure.